### ISSUE TITLE
How to open spot in C#

### ISSUE THREAD

**Issue by IvanKyuchukov** [gateio/gateapi-csharp#6]
I trying to make basic program that can open and close spots via API, I currently using your C# sample code, but I can't make it to work.
I downloaded the project, but there demo files weren't inside so I make new files for them, but now I get many errors in the RunConfig and SpotDemo files. Am I doing something wrong, aren't this files supposed to be inside the project?

**Comment by revilwang** [comment]
Did you try to start a new project and copy all the example codes to the new project. As for the SDK, did you try to use NuGet to retrieve this package?

**Comment by IvanKyuchukov** [comment]
Yes, I started new project, added all dependencies that were in the example project, also I added commandlineparser, because it wasn't in the main project, but RunConfig.cs using this. As for the SDK I am not very sure, now I will try to add it, via the OpenAPI Generator linked in the C# project, but I am not very sure for this step

**Comment by revilwang** [comment]
This SDK is also pushed to [NuGet](https://www.nuget.org/packages/Io.Gate.GateApi/). Try 

```
Install-Package Io.Gate.GateApi -Version 4.21.2
```

**Comment by IvanKyuchukov** [comment]
I installed it and when removed API, Model and Client folders now the aplication can start, but I am not sure do I need this folders, because now I get: 
ERROR(S):
  Required option 'k, key' is missing.
  Required option 's, secret' is missing.
  A required value not bound to option name is missing.

and I tried to put my Key and Secret inside, but still get error and even when change the key and secret value Required to false I still get " A required value not bound to option name is missing."
